docs/config/session.rst

```diff
@@ -276,4 +276,13 @@ consumption for the particular query.
   cr> SET search_path TO DEFAULT;
   SET OK, 0 rows affected (... sec)
 
+.. _conf-session-dml-fail-fast:
+
+**dml_fail_fast**
+  | *Default:* ``false``
+  | *Modifiable:* ``yes``
+   This setting enables partial failures of ``INSERT`` and ``UPDATE``
+   statements. If enabled, first error encountered gets propagated, operation
+   is terminated and number of inserted records is not reported.
```

Comment:
I'm not sure about the naming. I'd tend to highlight the partial write - e.g. `allow_fail_on_partial_writes`

Reply:
renamed in https://github.com/crate/crate/pull/17689/commits/41882d43fc9dd4773156b8d2c8a96f36ef0a5ebe